### PR TITLE
feat(swerve, intake, shooter): convert to AdvantageKit-compatible for…

### PR DESCRIPTION
**Pull request by TheGamer1002**
…mats, rewrite many parts

you can tell where it starts getting late :D

### CODE REVIEW

**Comment by BobcatNoah** [comment]
Oh man, this might take a while

**Comment by TheGamer1002** [comment]
lmao

**Comment by TheGamer1002** [comment]
honestly lets just see if it compiles

**Comment by TheGamer1002** [comment]
yoooo

**Comment by BobcatNoah** [comment]
Did it?

**Comment by TheGamer1002** [comment]
it does

**Comment by TheGamer1002** [comment]
im gonna go to bed i think

**Comment by TheGamer1002** [comment]
merge if you approve it please

**Comment by BobcatNoah** [comment]
I'm gonna help reviewing this. Please @NyxAlexandra

**Review by NyxAlexandra — CHANGES_REQUESTED** [review]
- related subsystems should be in the same module (ex: `Shooter`, `ShooterIO`)
- please get sleep, it's good for you :O

File: src/main/java/frc/robot/subsystems/shooter/ShooterSubsystem.java [hunk 1]
```diff
@@ -77,30 +59,34 @@ private ShooterSubsystem() {
...
 
 
   public void stop() {
     lowerMotor.set(0);
   }
+
+  /**
+   * Return the surface speed of the wheel in feet per second.
+   * @return the surface speed of the wheel in feet per second.
+   */
```
Comment (NyxAlexandra) [inline src/main/java/frc/robot/subsystems/shooter/ShooterSubsystem.java:86]: just do "The surface speed of the wheel in feet per second."

File: src/main/java/frc/robot/util/Constants.java [hunk 2]
```diff
@@ -15,24 +14,65 @@ public final class Constants {
...
     public static final double MAX_RPM = 5700;
+    public static final double MAX_VELOCITY = 2000;
+    public static final double MAX_ACCELERATION = 1500;
+
+
+    public static final double WHEEL_DIAMETER = 4; // inches
 
   }
 
+  public static final Mode currentMode = Mode.REAL;
```
Comment (NyxAlexandra) [inline src/main/java/frc/robot/util/Constants.java:36]: `CURRENT_MODE`

**Review by NyxAlexandra — DISMISSED** [review]

**Review by BobcatNoah — APPROVED** [review]

**Pull request merged**